db/migrations/0003_alter_order_user_alter_ticket_movie_session_and_more.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.2 on 2024-11-06 00:57
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_alter_movie_actors_alter_movie_genres_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='user',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='orders', to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='ticket',
+            name='movie_session',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tickets', to='db.moviesession'),
+        ),
+        migrations.AlterField(
+            model_name='ticket',
+            name='order',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tickets', to='db.order'),
```

Comment:
The `related_name='tickets'` for the `order` field in the `Ticket` model allows for reverse lookups from `Order` to `Ticket` instances. Ensure this related name is distinct within the model context.